ansible/roles/ocp4-workload-open-data-hub-olm/tasks/workload.yml

```diff
@@ -0,0 +1,32 @@
+# vim: set ft=ansible
+---
+
+# Implement your Workload deployment tasks here
+- name: Create Open Data Hub Operator Subscription
+  k8s:
+    state: present
+    namespace: "{{ operator_namespace }}"
+    definition: "{{ lookup('template', 'subscription.yaml.j2' ) }}"
+
+- name: Wait for the CRD to be available
+  k8s_facts:
+    api_version: "apiextensions.k8s.io/v1beta1"
+    kind: CustomResourceDefinition
+    name: "{{ item }}"
+  loop:
+    - kfdefs.kfdef.apps.kubeflow.org
+  register: crds
```

Comment:
We usually use the `r_` prefix on variable set via register. So this would be `r_crds`.

Reply:
Done. Out of curiosity, is there a reason for this? I'm not a huge Ansible guy and I'm curious if that's a standard convention or just in this project.